doc/RichTextArea/RichTextArea.md

```diff
@@ -0,0 +1,742 @@
+# Rich Text Area (Incubator)
+
+Andy Goryachev
+
+<andy.goryachev@oracle.com>
+
+
+## Summary
+
+Provide a RichTextArea control for displaying and editing of rich text that can be styled in a variety of ways.
+
+![RichTextArea screenshot](rich-text-area.png)
+
+
+
+## Goals
+
+Out of the box, the **RichTextArea** control provides support for a number of common use cases:
```

Comment:
I'm not sure if this was discussed before, do you think adding an example for each of the goals mentioned to show how it is supported is required? Or is it out of scope for this document?

Reply:
That's a great question.  Some of the less trivial examples **are** provided (key mappings, read-only models), many other are demonstrated with the demo code.  The purpose of this JEP is to provide a high level understanding of the new feature, with examples that illustrate less trivial concepts.

Which examples do you think should be added?

Reply:
You are right. Since examples like key mappings, read-only models are already provided, demo code would be sufficient to understand other goals and few of points are self explanatory which are basic features of rich text area. So the examples already provided would be sufficient.